features/docker/elasticsearch/elasticsearch.yml

```diff
@@ -0,0 +1,5 @@
+cluster.name: "docker-cluster"
+network.host: 0.0.0.0
+
+# You need to create this folder in order to run snapshots
+path.repo: ["/tmp/snapshots"]
```

Comment:
Can't we pass this variable to ES with the environment instead of having to build a whole new Docker image?

Reply:
I tried, does not work

Reply:
Why :thinking:

Reply:
I think this is due to the fact that the value is an array. Could you try to use YAML objects for environment variables? Like this:
```yaml
services:
  kuzzle:
  # ...
  elasticsearch:
    image: yourImage
    environments:
      path.repo:
        - /tmp/snapshots
    # ...
```
or using regular YAML array of string:
```yaml
services:
  kuzzle:
  # ...
  elasticsearch:
    image: yourImage
    environments:
      - "path.repo=[/tmp/snapshots]"
    # ...
```

Reply:
Double quote in the second snippet are required for YAML parsing

Reply:
I'll try that, get back to you if it works or not 👍

Reply:
@alexandrebouthinon both of the options you proposed weren't successful. Only option I see is to keep it that way :x

Reply:
Ok I thought it was a parsing and type inference problem. That's good for me then 🙂